src/coffee/directives/api/models/parent/poly-parent-models.coffee

```diff
@@ -0,0 +1,6 @@
+['Polygon','Polyline'].forEach (name) ->
```

Comment:
The factories are basically the same but differ by name to different gObject types.